src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const employeesList = [...document.querySelectorAll('[data-position]')];
+
+function sortList(items) {
+  const sortedList = items.sort((a, b) => {
+    const salaryA = parseInt(a.dataset.salary.replace(/[$,]/g, ''), 10);
+    const salaryB = parseInt(b.dataset.salary.replace(/[$,]/g, ''), 10);
+
+    return salaryB - salaryA;
+  });
+
+  const parentElem = items[0].parentElement;
+
+  parentElem.innerHTML = '';
+
+  sortedList.forEach((el) => {
+    parentElem.appendChild(el);
+  });
+}
```

Comment:
The `sortList` function sorts the elements based on their salary data and updates the DOM. Ensure that the elements have the `data-salary` attribute, as missing attributes could lead to `NaN` values during parsing.